packages/eslint-config-meilisearch/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "eslint-config-meilisearch",
+  "version": "0.0.0",
```

Comment:
Since we are not publishing it, its not needed. See example from turbo's directory: https://github.com/vercel/turbo/blob/main/examples/basic/packages/eslint-config-custom/package.json